apps/data/tests/io/test_cgd.py

```diff
@@ -113,6 +120,15 @@ def test__to_entries_with_existing_entries(self):
         )
         assert parser.entries['update'][0].url == first_entry.url
 
+    def test__to_entries_with_aliases(self):
+
+        parser = ChrFeatureParser(self.file_path)
+        parser.parse()
+        parser._to_entries(ignore_aliases=False)
+
+        assert len(parser.entries['new']) == 30
```

Comment:
Where does `30` come from?

Reply:
30 is the sum of identifiers (10) and unique aliases (20) in `apps/data/fixtures/C_glabrata_CBS138_current_chromosomal_feature_10.tab`